packages/core/src/overlay/OverlayPanelCloseButton.css

```diff
@@ -9,7 +9,7 @@
   right: 0;
   top: 0;
 }
-
-.saltOverlayPanelCloseButton ~ .saltOverlayPanelContent {
+.saltOverlayPanelCloseButton ~ .saltOverlayPanelContent,
+.saltOverlayPanelCloseButton ~ .saltOverlayHeader {
```

Comment:
Is this backwards compatibility check? This will move existing `OverlayPanelCloseButton`? I'd guess we don't want to move existing visual, unless people opt to use `actions` api..? Or should this be added to dialog?